src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -31,7 +31,7 @@
   </div>
 
   <div class="card-body">
-    <div class="info-block">
+    <div id="customerBasicInformations" class="info-block">
```

Comment:
Please don't use id when the block can be displayed more than once in a page. Don't forget this file is a customer card.

Reply:
Eh ?
This file is `Order/Blocks/View/customer.html.twig`

It's supposed to be displayed once, in the Order page 🤔  else it's not in the right folder

Reply:
It's in the `Blocks` directory, included in a `view.html.twig` file :thinking: 
For anyone familiar with the block system in templates (django, twig, etc), this means you can inject this block more than once :thinking: 
Even if "it's supposed" it doesn't have to be that way, maybe a module developer will use this file more than once. It costs nothing to use classes when it's possible.